### PR TITLE
Bump quarkus version from 2.5.2 to 2.11.1

### PR DESCRIPTION
**Pull request by vbusch**
Signed-off-by: Vanessa Busch <vbusch@redhat.com>

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
I think we should bumpto the latest version - 2.11.1 or something. Not sure there is much valie in bumping it to version which is already old.

**Comment by vbusch** [comment]
> I think we should bumpto the latest version - 2.11.1 or something. Not sure there is much valie in bumping it to version which is already old.

I have updated it to 2.11.1

**Review by scholzj — APPROVED** [review]
LGTM. Thanks.

File: pom.xml [hunk 1]
```diff
@@ -37,6 +38,46 @@
...
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-junit5-mockito</artifactId>
+        <version>${quarkus.test.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-junit5</artifactId>
+        <version>${quarkus.test.version}</version>
+      </dependency>
```
Comment (ppatierno) [inline pom.xml:80]: @vbusch out of curiosity, why the addition of all these dependencies not needed before? Why for example the ones related to junit5 and mockito are not "test" scoped?

Comment (vbusch) [reply pom.xml]: @ppatierno thank you.  I was able to remove them.

**Review by ppatierno — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR @vbusch

**Pull request merged**